api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -639,6 +663,58 @@ def test_github_webhook_delete_installation(
     assert not GithubConfiguration.objects.filter(installation_id=1234567).exists()
 
 
+def test_github_webhook_merged_a_pull_request(
+    feature: Feature,
+    github_configuration: GithubConfiguration,
+    github_repository: GithubRepository,
+    feature_external_resource: FeatureExternalResource,
+) -> None:
+    # Given
+    settings.GITHUB_WEBHOOK_SECRET = WEBHOOK_SECRET
+    url = reverse("api-v1:github-webhook")
+
+    # When
+    client = APIClient()
```

Comment:
Why not use the `api_client` fixture instead?

Reply:
Done